src/next-child-view-container.js

```diff
@@ -0,0 +1,108 @@
+import _ from 'underscore';
+import emulateCollection from './utils/emulate-collection';
+
+// Provide a container to store, retrieve and
+// shut down child views.
+const Container = function() {
+  this._views = [];
```

Comment:
There are some things about it that I notice that will not work so well for the child rendering, such as https://github.com/joefitter/backbone.enumerable/blob/master/src/backbone.enumerable.js#L100

which will prevent more than one child constructor type from being added to the list.

I think adopting some of the logic from `Backbone.Enumerable` is best.